pkgs/by-name/es/esp-generate/package.nix

```
@@ -0,0 +1,22 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "esp-generate";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "esp-rs";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-vfHORu0C3vK7a9jv1Wj+XjqM7mR8/t0N4x3hfARdxaw=";
+  };
+
+  cargoHash = "sha256-hYgEZvRXXhgzuNuSu4ytrlWDwlVYX6R46agu+BxrbEo=";
+
+  meta = {
+    description = "Template generation tool to create no_std applications targeting Espressif's chips";
+    homepage = "https://github.com/BurntSushi/ripgrep";
+    license = lib.licenses.mit;
```

```suggestion
    license = with lib.licenses; [ mit /* or */ asl20 ];
```
Copied from espflash:
https://github.com/NixOS/nixpkgs/blob/dc460ec76cbff0e66e269457d7b728432263166c/pkgs/by-name/es/espflash/package.nix#L59